### PR TITLE
Delete rudiment widgets

### PR DESCRIPTION
**Pull request by dimas9009**
1. Added step to delete rudiment widgets.
2. Added file to describe allowed widgts in final mpk

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -94,6 +94,36 @@ jobs:
...
+            fi
+
+            if [ -d "$TEMP_UNZIP_FOLDER" ]; then rm -Rf $TEMP_UNZIP_FOLDER; fi
+            mkdir $TEMP_UNZIP_FOLDER
+            unzip -x $TARGET_MPK -d $TEMP_UNZIP_FOLDER
+
+            cd "$TEMP_UNZIP_FOLDER/widgets"
+            for i in *; do
+              if ! grep -q "$i" $ALLOWED_LIST; then
+                echo "Deleting widget: $i" | tee $GITHUB_WORKSPACE/out/deletedWidgets.log
```
Comment (allard-mx) [inline .github/workflows/build.yml:115]: Currently the deletedWidgets.log file seems to be overwritten for every widget that gets deleted. In the results, the file currently only contains 'Deleting widget: com.mendix.widget.web.DatagridTextFilter.mpk', which is the latest widget that got deleted.

**Review by allard-mx — CHANGES_REQUESTED** [review]
Seems to work really well. Added one review comment.

**Review by allard-mx — CHANGES_REQUESTED** [review]
We found one other issue: the package.xml is currently not updated. This results in Studio Pro giving a warning that the removed widget files will still be imported, even though this is not the case.

**Review by allard-mx — APPROVED** [review]

**Review by MendixMurshed — APPROVED** [review]
great, working nicely

**Pull request merged**